.env.example

```diff
@@ -0,0 +1,6 @@
+This is a template .env file that we can add to as we start using environment variables. 
```

Comment:
Nit: please put these in as [comments](https://akhromieiev.com/how-to-add-comment-in-env-file/)